compiler/rustc_middle/src/mir/coverage.rs

```diff
@@ -311,10 +297,9 @@ pub struct MCDCBranchSpan {
     pub span: Span,
     /// If `None`, this actually represents a normal branch span inserted for
     /// code that was too complex for MC/DC.
```

Comment:
This comment seems inaccurate now; should it be removed?

Reply:
Sure it is removed at [compare](https://github.com/rust-lang/rust/compare/541e9a280ad3c73ebdd5230d3ec1f1af5a83d8b6..473131a57bc6cffd88272a22b384a9f08b46ab16)